cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/OperatorsTest.scala

```diff
@@ -245,9 +253,96 @@ class OperatorsTest extends DocumentingTest {
           |In addition, refer to <<query-operator-comparison-string-specific>> for details on string-specific comparison operators.""")
     }
+    section("Temporal operators", "query-operators-temporal") {
+      p(
+        """Temporal operators comprise:
+          |
+          |* adding a <<cypher-temporal-durations, _Duration_>> to either a <<cypher-temporal-instants, temporal instant>> or another _Duration_: `+`
+          |* subtracting a _Duration_ from either a temporal instant or another _Duration_: `-`
+          |* multiplying a _Duration_ with a number: `*`
+          |* dividing a _Duration_ by a number: `/`
+          |
+          |The following table shows -- for each combination of operation and operand type -- the type of the value returned from the application of each temporal operator:
+          |
+          |[options="header"]
+          ||===
+          || Operator | Left-hand operand | Right-hand operand | Type of result
+          || <<syntax-add-subtract-duration-to-temporal-instant, `+`>> | Temporal instant           | _Duration_                 | The type of the temporal instant
+          || <<syntax-add-subtract-duration-to-temporal-instant, `+`>> | _Duration_                 | Temporal instant           | The type of the temporal instant
+          || <<syntax-add-subtract-duration-to-temporal-instant, `-`>> | Temporal instant           | _Duration_                 | The type of the temporal instant
+          || <<syntax-add-subtract-duration-to-duration, `+`>>         | _Duration_                 | _Duration_                 | _Duration_
+          || <<syntax-add-subtract-duration-to-duration, `-`>>         | _Duration_                 | _Duration_                 | _Duration_
+          || <<syntax-multiply-divide-duration-number, `*`>>           | _Duration_                 | <<property-types, Number>> | _Duration_
+          || <<syntax-multiply-divide-duration-number, `*`>>           | <<property-types, Number>> | _Duration_                 | _Duration_
+          || <<syntax-multiply-divide-duration-number, `/`>>           | _Duration_                 | <<property-types, Number>> | _Duration_
+          ||===
+          |
+        """)
+      section("Adding and subtracting a _Duration_ to or from a temporal instant", "syntax-add-subtract-duration-to-temporal-instant") {
+        query(
+          """WITH datetime({year:1984, month:10, day:11, hour:12, minute:31, second:14, nanosecond: 1, timezone: '+01:00'}) AS aDateTime
+            |UNWIND [duration({years: 12, months: 5, days: 14, hours: 16, minutes: 12, seconds: 70, nanoseconds: 2}),
+            |   duration({months:1, days: -14, hours: 16, minutes: -12, seconds: 70}),
+            |   duration({years: 12.5, months: 5.5, days: 14.5, hours: 16.5, minutes: 12.5, seconds: 70.5, nanoseconds: 3})] AS aDuration
+            |RETURN aDateTime + aDuration, aDateTime - aDuration""".stripMargin, ResultAssertions((r) => {
```

Comment:
I think the queries should return the original date and duration fields, otherwise it is hard to understand the results.